README.md

```diff
@@ -28,7 +28,7 @@ Both liboqs and this fork are part of the **Open Quantum Safe (OQS) project**, w
 
 ## Status
 
-This fork is built on top of [commit 5bcb626](https://github.com/google/boringssl/commit/5bcb626c847a10e2e631118b637c9db25593cdea), and adds:
+This fork is built on top of [commit f10c1dc](https://github.com/google/boringssl/commit/f10c1dc37174843c504a80e94c252e35b7b1eb61), and adds:
```

Comment:
Not quite right any more as this does not add QSC KEMs in general any more but just some that are not already in boringssl...

Reply:
updated

Reply:
I've updated readme

Reply:
Thanks @pi-314159 Your change does require people to know which algorithms Google implemented, though. Also there is no warning at the KEX-hybrid section below that different implementations are used. 

@dstebila can we please discuss this at an upcoming OQS meeting? Question: How to deal with upstream projects that begin adding PQC bit-by-bit? It may just be some communication (e.g., whom to contact in which situation) but also may require some clear-cut policy. This is not to ask for supremacy of code coming from OQS (or from any PQCA project in general) -- to the contrary: We should always support the best code -- but be clear about what "best" means and which code it is in each case. For this PR, I'm all for supporting the Google code as it implements the more current standards -- even at the cost of (eventually, possibly) supporting code written by the libjade team: Standards compliance and availability should trump arguably conceptual security advantages -- but again, open to debate.

Reply:
1. In line 69 of [this commit](https://github.com/open-quantum-safe/boringssl/pull/122/commits/5918e7dc58ffb57d234343cd47b56faa9e5707e7), I clearly stated that `X25519MLKEM768` and `X25519Kyber768Draft00` are natively supported by BoringSSL. Additionally, I noted that for algorithms natively supported by BoringSSL, Google's implementation is used. Therefore, users should expect these two algorithms are implemented by Google, not by liboqs.

2. Overall, BoringSSL and OQS-BoringSSL do not guarantee API stability, as indicated in line 40 of the README.md. These libraries are intended for use in projects like Chromium and QUIC. The README explicitly mentions, "It works for us, but it may not work for you," highlighting that it's not designed for general public use like OpenSSL. Therefore, I'll leave the situation as it is.

Reply:
1. In line 69 of [this commit](https://github.com/open-quantum-safe/boringssl/pull/122/commits/5918e7dc58ffb57d234343cd47b56faa9e5707e7), I clearly stated that `X25519MLKEM768` and `X25519Kyber768Draft00` are natively supported by BoringSSL. Additionally, I noted that for algorithms natively supported by BoringSSL, Google's implementation is used. Therefore, users should expect these two algorithms are implemented by Google, not by liboqs.

2. Overall, BoringSSL and OQS-BoringSSL do not guarantee API stability, as indicated in line 40 of the README.md. These libraries are intended for use in projects like Chromium and QUIC. The README explicitly mentions, "It works for us, but it may not work for you," highlighting that it's not designed for general public use like OpenSSL...

Reply:
> In line 69 of [this commit](https://github.com/open-quantum-safe/boringssl/pull/122/commits/5918e7dc58ffb57d234343cd47b56faa9e5707e7), I clearly stated that X25519MLKEM768 and X25519Kyber768Draft00 are natively supported by BoringSSL.

I saw that but also still see the generic statement below

> For each <KEX> listed above, the following hybrid algorithms are made available as follows

I guess what I'm missing is a clear statement at that place to the tune of "where both are available, the PQC code of Google will be executed instead of the OQS code".

> These libraries are intended for use in projects like Chromium and QUIC. The README explicitly mentions, "It works for us, but it may not work for you," highlighting that it's not designed for general public use like OpenSSL

ACK. Indeed I overlooked those: They are clear enough. I'm then fine with the PR as-is but still recommend a general review by the OQS team how to deal with such "dual-origin" PQC software stacks (@dstebila @SWilson4 @praveksharma ).

Reply:
I'll change the algs list in README.md later